wire-kotlin-generator/src/test/java/com/squareup/wire/kotlin/KotlinGeneratorTest.kt

```diff
@@ -276,8 +276,8 @@ class KotlinGeneratorTest {
           |interface RouteGuideBlockingServer : Service {
```

Comment:
Remove the comment line 269

Reply:
`@WireRpc` _path_ doesn’t use the Java package, but the adapters do!

Reply:
hmmmm